acapy/Dockerfile.acapy

```diff
@@ -14,6 +14,10 @@ USER $user
 COPY acapy/start.sh start.sh
 COPY acapy/configs configs
 
+# Copy and install plugins
+COPY acapy/plugins plugins
+RUN pip3 install --no-cache-dir -e plugins/firebase_push_notifications
```

Comment:
I think it would be better to keep packages in a `requremetns.txt` file so they're more obvious and remove caches when done to keep the image smallish.

```
COPY requirements.txt /tmp/pip-tmp/

RUN /usr/bin/pip install \
  --disable-pip-version-check \
  --no-cache-dir \
  --trusted-host=pypi.python.org \
  --trusted-host=pypi.org \
  --trusted-host=files.pythonhosted.org \
  -r /tmp/pip-tmp/requirements.txt && \
  rm -rf /tmp/pip-tmp
  ```